loadbalancer.go

```diff
@@ -732,3 +731,154 @@ func (p *pool) Delete(ctx context.Context, id string) error {
 
 	return resp.Body.Close()
 }
+
+type HealthMonitor struct {
+	Name           string                `json:"name"`
+	Type           string                `json:"type"`
+	Delay          int                   `json:"delay"`
+	MaxRetries     int                   `json:"max_retries"`
+	MaxRetriesDown int                   `json:"max_retries_down"`
+	TimeOut        int                   `json:"timeout"`
+	HTTPMethod     string                `json:"http_method"`
+	UrlPath        string                `json:"url_path"`
+	ExpectedCodes  string                `json:"expected_codes"`
+	HTTPVersion    float32                `json:"http_version"`
+	OpratingStatus string                `json:"oprating_status"`
+	DomainName     string                `json:"domain_name"`
+	ID             string                `json:"id"`
+	CreatedAt      string                `json:"created_at"`
+	UpdatedAt      string                `json:"updated_at"`
+	TenantID       string                `json:"tenant_id"`
+	Pool           []struct{ ID string } `json:"pool"`
```

Comment:
This `[]struct{ ID string }` is used in many places, maybe we should add new type for this dedicated type.

Reply:
Yes.